share/spice/cricket/cricket.js

```diff
@@ -0,0 +1,159 @@
+                    scrollToSelectedItem: true,
+                    itemsHighlight: true,
+                    itemsExpand: true,
+                    //secondaryText: '<span class="tx-clr--grey-dark">YQL</span>',
+                    rerender: ["teams.0", "teams.1"],//TODO: need a better way
+                    //sourceName: "developer.yahoo.com",
+                    //sourceUrl: 'http://example.com/url/to/details/' + api_result.name
+                },
+                normalize: function (item) {
+                    var winner = getWinner(item);
+                    return {
+                        matchid: item.matchid,
+                        mtype: item.mtype.toUpperCase(),
+                        matchNo: item.MatchNo,
+                        venue: item.Venue.content,
+                        date: moment.utc(item.StartDate).local().format('ddd, Do MMM'),
+                        relativeTime: moment.utc(item.StartDate).local().calendar(),
+                        startTime: moment.utc(item.StartDate).local().format('LT'),
+                        startDate: Date.parse(moment.utc(item.StartDate).local().format("YYYY-MM-DD")),
+                        matchTimeSpan: item.MatchTimeSpan,
+                        teams: item.Team,
+                        result: item.Result,
+                        winnersn: winner && winner.split(" ").length > 1 ? winner.match(/\b(\w)/g).join('') : winner,
+                        winnerln: winner,
+                    };
+                },
+                templates: {
+                    item: "base_expanding_item",
+                    elClass: {
+                        tileExpand: "zci-cricket-footer"
+                    },
+                    options: {
+                        content: Spice.cricket.content,
+                        moreAt: false
+                    }
+                },
+                onItemShown: function (item) {
+                    item.teams.map(function (team, i) {
+                        $.ajax({
+                            url: DDG.get_asset_path('cricket', "assets/" + team.teamid + ".png"),
+                            type: 'get',
+                        }).always(function (data, statusText, xhr) {
+                            // Check if team logo exists
+                            if (xhr.status === 200 && xhr.getResponseHeader('content-type') == "image/png") {
+                                team.logo = DDG.get_asset_path('cricket', "assets/" + team.teamid + ".png");
+                                //TODO: need a better way
+                                item.set("teams." + i, item.teams);
+                            } else {
+                            }
+                        });
+                        return team;
+                    });
+                    fetchScore(item.matchid)
+                }
+
+            });
+        });
+        //TODO: return latest match id
+        function getSelected(matches) {
```

Comment:
Why not define this within the `require()` above and use Moment?

You can also wrap this in another `require()` if you really want